spec/ics-006-solo-machine-client/README.md

```diff
@@ -47,12 +47,13 @@ interface ClientState {
 
 ### Consensus state
 
-The `ConsensusState` of a solo machine consists of the current public key & sequence number.
+The `ConsensusState` of a solo machine consists of the current public key, current diversifier, sequence number, and timestamp.
 
 ```typescript
 interface ConsensusState {
   sequence: uint64
   publicKey: PublicKey
+  diversifier: string
```

Comment:
what's a diversifier? can you add a comment here for future reference? 👍

Reply:
Yes - a diversifier is a string chosen on client creation which allows a public-key to be reused for multiple solo clients

https://github.com/cosmos/ics/commit/aaa16eb0792c126a4bf8d7ed1d919d9e75b4bb99